setup.py

```diff
@@ -29,10 +29,11 @@
         'cfscrape>=2.0.5',
         'requests-cache>=0.4.13',
         'tabulate>=0.8.3',
-        'pycryptodome>=3.8.2',
+
     ],
     extras_require={
         'selescrape': ['selenium'],
+        'pycryptodome>=3.8.2',
```

Comment:
You should do `'key': 'value'` as it is in a dict, and will cause errors. I recommend something like: `'crypto': ['pycryptodome>=3.8.2']`